snmp.yml

```diff
@@ -2962,7 +3029,6 @@ modules:
       indexes:
       - labelname: bsnAPDot3MacAddress
         type: PhysAddress48
-        fixed_size: 6
```

Comment:
this should not have happend - do you remember how you ran the generator and which version it was?

Reply:
added the mibs files to the generator/mibs

make generator mibs
make generate

./generator generate

copy the generator/snmp.yml to ../snmp.yml

Reply:
This generally indicates a miss-match problem with net-snmp. We should have fixed issue was fixed a while ago with some CGO workarounds. However, I think some MacOS users were still seeing issues.